management/src/main/java/org/ehcache/management/cluster/DefaultClusteringManagementService.java

```diff
@@ -40,20 +41,37 @@
 import org.terracotta.management.model.stats.ContextualStatistics;
 
 import java.util.Collection;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ExecutorService;
 
-@ServiceDependencies({CacheManagerProviderService.class, ExecutionService.class, TimeSourceService.class, ManagementRegistryService.class, EntityService.class})
+@ServiceDependencies({CacheManagerProviderService.class, ExecutionService.class, TimeSourceService.class, ManagementRegistryService.class, EntityService.class, ExecutionService.class})
```

Comment:
That's twice the same dependency here - I know this was merged but please clean it up.